### ISSUE TITLE
SNES MAX gamepad support - raphnet 2x adapters

### ISSUE THREAD

**Issue by Eriknoc** [AppleWin/AppleWin#1142]
Both adapters are from www.raphnet.com.  They both follow the same button numbered order, except the NES obviously doesn't use any buttons beyond 4.

Dual SNES to USB v2.2 / ADAP-2XSNES-USB_v2 / p/n: 6-039
Buttons: 1=Y, 2=B, 3=Select, 4=Start, 5=A, 6=X, 7=Lb, 8=Rb

Dual NES to USB v2.2 / ADAP-2XNES-USB_v2 / p/n: 6-034
Buttons: 1=B, 2=A, 3=Select, 4=Start

**Comment by Eriknoc** [comment]
Are there any property details I can get for you?  Does any of this help?
[HID-compliant Hardware Ids.txt](https://github.com/AppleWin/AppleWin/files/10027607/HID-compliant.Hardware.Ids.txt)

**Comment by tomcw** [comment]
Yes - how did you generate that .txt file?

**Comment by Eriknoc** [comment]
I created the text file myself and did copy and past for  group.  While in the properties, I went into each Hardware Id one by one and I right-clicked then clicked "select all", then right-clicked on the selected text and clicked "copy" and pasted into my text file.  I then gave it a name base on what each group had in common, which was HID-compliant xxx and the sub-detail of Hardware Id.  And of course, I had to show which ones were for the SNES and which ones were for the NES.  We don't want to mix those up.  :-)
